tests/test_mock_controller.py

```diff
@@ -3,6 +3,7 @@
 from asynctest.mock import CoroutineMock
 import logging
 import math
+import numpy as np
```

Comment:
Picky point, but please insert a blank line between standard library and 3rd party imports

Reply:
Thanks, I keep forgetting about that.

Reply:
Done